Sources/News.php

```diff
@@ -233,8 +233,8 @@ function ShowXmlFeed()
 			'dc' => 'http://purl.org/dc/elements/1.1/',
 		),
 		'smf' => array(
-			'' => 'http://www.simplemachines.org/xml/' . $_GET['sa'],
-			'smf' => 'http://www.simplemachines.org/',
+			'' => 'https://www.simplemachines.org/xml/' . $_GET['sa'],
+			'smf' => 'https://www.simplemachines.org/',
```

Comment:
Since they are namespace declarations rather than linkable URLs, these should stay http. In reality, most XML processors aren't going to recognize SMF's proprietary namespace, so in real world applications this change wouldn't make much difference. But to be standards compliant, we should keep this namespace exactly as it has always been defined.